python/src/FunctionalChaosRandomVector_doc.i.in

```diff
@@ -9,10 +9,102 @@ Parameters
 functionalChaosResult : :class:`~openturns.FunctionalChaosResult`
     A result from a functional chaos decomposition.
 
+Notes
+-----
+This class can be used to get probabilistic properties of a
+functional chaos expansion or polynomial chaos expansion (PCE).
+For example, we can get the output mean or the output covariance matrix
+using the coefficients of the expansion.
+
+Moreover, we can use this class to simulation random observations
+of the output.
+We consider the same notations as in the :class:`~openturns.FunctionalChaosAlgorithm`
+class.
+The functional chaos decomposition of *h* is:
+
+.. math::
+
+    h = g\circ T^{-1} = \sum_{k=0}^{\infty} \vect{a}_k \Psi_k 
+
+
+which can be truncated to the finite set :math:`\cK \subset \Nset`:
+
+.. math::
+
+    \widetilde{h} =  \sum_{k \in \cK} \vect{a}_k \Psi_k 
+
+The approximation :math:`\widetilde{h}` can be used to build an efficient random 
+generator of :math:`Y` based on the random vector :math:`\vect{Z}`,
+using the equation:
+
+.. math::
+
+    \widetilde{Y} = \widetilde{h}(\vect{Z})
+
+This equation can be used to simulate independent random observations
+from the PCE.
+This can be done by first simulating independent observations from
+the distribution of the standardized random vector $\vect{Z}$,
+then push forward these observations through the expansion.
```

Comment:
```suggestion
This can be done by simulating independent observations from
the distribution of the standardized random vector :math:`\vect{Z}`,
which are then pushed forward through the expansion.
```